anaDACScan.py

```diff
@@ -288,13 +288,11 @@
             #evaluate the fitted function at the nominal current or voltage value and convert to an integer
             nominalDacValue = int(dict_DACvsADC_Funcs[oh][vfat].Eval(nominal))
             
-            if nominalDacValue > maxDacValue:
-                print('Warning: the nominal DAC value that was found from the fit is larger than the maximum value that the DAC register will hold')
+            if nominalDacValue < 0 or nominalDacValue > maxDacValue:
+                print('Warning: The fitted DAC value, '+str(nominalDacValue)+', is outside of the range that the register can hold: [0,'+str(maxDacValue)+']. It will be replaced by '+str(max(0,min(maxDacValue,nominalDacValue)))+'.')
+                nominalDacValue = max(0,min(maxDacValue,nominalDacValue))
```

Comment:
If you reverse L293 and L292, and use string formatting (rather than concatenation), the long term maintainability will improve

Reply:
ok, changed

@bdorney, can you confirm that 0 is always the minimum DAC value?

Reply:
For the DAC's that are considered the minimum is 0, see:

https://github.com/cms-gem-daq-project/ctp7_modules/blob/4d82a717400fd96cb6f18104b58b93a1e8751036/src/calibration_routines.cpp#L1372-L1401